wezterm-gui/src/termwindow/mod.rs

```diff
@@ -1432,6 +1436,14 @@ impl TermWindow {
         };
 
         if let Some(window) = self.window.as_ref().map(|w| w.clone()) {
+            self.os_parameters = match window.get_os_parameters(&config) {
+                Ok(os_parameters) => os_parameters,
+                Err(_) => {
+                    log::warn!("Error while getting OS parameters");
```

Comment:
It's almost always a good idea to include context in error messages!

```suggestion
                Err(err) => {
                    log::warn!("Error while getting OS parameters: {:#}", err);
```